src/Core/Migration/V6_7/Migration1752219159AddLanguageActive.php

```diff
@@ -0,0 +1,53 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Core\Migration\V6_7;
+
+use Doctrine\DBAL\Connection;
+use Shopware\Core\Framework\Log\Package;
+use Shopware\Core\Framework\Migration\MigrationStep;
+
+/**
+ * @internal
+ */
+#[Package('framework')]
+class Migration1752219159AddLanguageActive extends MigrationStep
+{
+    public function getCreationTimestamp(): int
+    {
+        return 1752219159;
+    }
+
+    public function update(Connection $connection): void
+    {
+        $addedColumn = $this->addColumn(
+            $connection,
+            table: 'language',
+            column: 'active',
+            type: 'TINYINT(1)',
+            nullable: false,
+            default: '0',
+        );
+
+        if ($addedColumn) {
+            $connection->executeStatement(
+                <<<'SQL'
+                    UPDATE `language`
+                    SET `active` = 1
```

Comment:
is that intended, that we activate all languages by default? :thinking:

Reply:
Yes, since before it was like "If we have a language, it counts as activated, if you don't want it, delete it". The change does not add any language, but makes sure they work as intended, when they're already installed.